build/BuildSteps.cs

```diff
@@ -583,5 +588,17 @@ private static List<string> GetV3PackageList()
 
             return packageList;
         }
+
+        private static void RemoveLanguageWorkers(string outputPath)
```

Comment:
I realize this is a tactical fix, but if we're not adding tests here, or intend to change the approach, can we please have an issue filed and linked for follow up?

Reply:
Yes absolutely. I will be opening an issue with proper details for a follow up.

Reply:
https://github.com/Azure/azure-functions-core-tools/issues/2380